packages/flutter_tools/test/integration.shard/deprecated_gradle_settings_test.dart

```diff
@@ -16,6 +17,40 @@ void main() {
     final String workingDirectory = fileSystem.path.join(getFlutterRoot(), 'dev', 'integration_tests', 'gradle_deprecated_settings');
     final String flutterBin = fileSystem.path.join(getFlutterRoot(), 'bin', 'flutter');
 
+    final File settingsDotGradleFile = fileSystem.file(
+        fileSystem.path.join(workingDirectory, 'android', 'settings.gradle'));
+    const String settingsDotGradle = r"""
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This is the `settings.gradle` file that apps currently use.
```

Comment:
This comment is a confusing. It is in a deprecated_gradle_settings_test but states the setting is current but also declares it must be migrated without pointing to any information that would help the author migrate.

Reply:
I moved this to #140037